src/components/InputBook.js

```diff
@@ -30,22 +30,27 @@ const InputBook = () => {
 
   return (
     <form
+      className="input-book"
       onSubmit={handleSubmit((event) => {
         addNewBook(event);
       })}
     >
       {/* eslint-disable react/jsx-props-no-spreading */}
       <input
-        {...register('title', { required: true })}
-        placeholder="name of the book"
+        className="input"
+        {...register('title', { required: true && (<span />) })}
+        placeholder="Book title"
       />
       {errors.title?.type === 'required' && 'Title is required'}
       <input
+        className="input"
         {...register('author', { required: true })}
-        placeholder="author of the book"
+        placeholder="Author"
       />
       {errors.author?.type === 'required' && 'Author is required'}
-      <button type="submit">Add book</button>
+      <button className="update-button" type="submit">
+        Add book
+      </button>
```

Comment:
- [ ] Let's also make the button as similar as the one from the template, including the correct fonts :tada:

Reply:
Ok